test/jit/test_torchbind.py

```diff
@@ -130,6 +130,67 @@ def foo():
         self.assertEqual(out[1].pop(), "mom")
         self.assertEqual(out[1].pop(), "hi")
 
+    def test_torchbind_def_property_getter_setter(self):
+        def foo_getter_setter_full():
+            fooGetterSetter = torch.classes._TorchScriptTesting._FooGetterSetter(5, 6)
+            # getX method intentionally adds 2 to x
+            old = fooGetterSetter.x
+            # setX method intentionally adds 2 to x
+            fooGetterSetter.x = old + 4
+            new = fooGetterSetter.x
+            return old, new
+
+        self.checkScript(foo_getter_setter_full, ())
+
+        def foo_getter_setter_lambda():
+            foo = torch.classes._TorchScriptTesting._FooGetterSetterLambda(5)
+            old = foo.x
+            foo.x = old + 4
+            new = foo.x
+            return old, new
+
+        self.checkScript(foo_getter_setter_lambda, ())
+
+    def test_torchbind_def_property_just_getter(self):
+        def foo_just_getter():
+            fooGetterSetter = torch.classes._TorchScriptTesting._FooGetterSetter(5, 6)
+            # getY method intentionally adds 4 to x
+            return fooGetterSetter, fooGetterSetter.y
+
+        scripted = torch.jit.script(foo_just_getter)
+        out, result = scripted()
+        self.assertEqual(result, 10)
+
+        with self.assertRaisesRegex(RuntimeError, 'can\'t set attribute'):
```

Comment:
Consider using `self.assertRaisesRegexWithHighlight` here (and in other places where `assertRaisesRegex` has been used) to make sure the correct portion of the source code is highlighted.